packages/email-templates/src/index.ts

```diff
@@ -63,4 +63,4 @@ Object.keys(EmailTemplates).forEach((template: string) => {
 const getEmailTemplate = (template: Template | string) => templates[template]
 
 export default templates
-export { getEmailTemplate, base }
+export { getEmailTemplate, bases }
```

Comment:
I'm a little worried about this change since base is not under bases which may break something.

Reply:
Not sure what you mean?

Reply:
the exports are changed. If you want to import base now, you have to import it from under bases.

```ts
import { bases } from "./templates.ts"
const { base } = bases
```